Library/Homebrew/brew.sh

```diff
@@ -336,7 +346,12 @@ else
   # shellcheck source=/dev/null
   source "$HOMEBREW_LIBRARY/Homebrew/utils/ruby.sh"
   setup-ruby-path
-
+  
+  if [[ -n "$HOMEBREW_WHICH_RUBY" ]]
```

Comment:
OK a little bit a dev history left in there.

I thought that it would be useful to add the debugger to the 'cmd/install.rb' to see what ENV it was getting after the various Homebrew additions but the gem couldn't be found originally due to using 'vendor/portable-ruby' (which gets updated if changed) or the '/System' one (due to being a Mac) but what I really wanted was my RVM one with the installed 'byebug' gem (also the reason for leaving in the 'GEM_*' since my dev env was to drop into a debugger shell on install).

I left it in due to the ability to use a debugger must be useful to other devs.